onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportTypeListFragment.java

```diff
@@ -0,0 +1,162 @@
+            reportIcons = getResources().obtainTypedArray(R.array.report_types_icons);
+        } else {
+            reportTypes = getResources().getStringArray(R.array.report_types_without_open311);
+            reportDesc = getResources().getStringArray(R.array.report_types_desc_without_open311);
+            reportIcons = getResources().obtainTypedArray(R.array.report_types_icons_without_open311);
+        }
+
+        Boolean isEmailDefined = isEmailDefined();
+        List<ReportTypeItem> reportTypeItems = new ArrayList<>();
+        for (int i = 0; i < reportTypes.length; i++) {
+            //Don't show the send app feedback section if email is not defined for region
+            if (!isEmailDefined && getString(R.string.rt_app_feedback).equals(reportTypes[i]))
+                continue;
+            ReportTypeItem item = new ReportTypeItem(reportTypes[i], reportDesc[i], reportIcons.getResourceId(i, -1));
+            reportTypeItems.add(item);
+        }
+
+        ReportTypeListAdapter adapter = new ReportTypeListAdapter(getActivity(), reportTypeItems);
+        setListAdapter(adapter);
+        getListView().setOnItemClickListener(this);
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        ObaAnalytics.reportFragmentStart(this);
+    }
+
+    private Boolean isEmailDefined() {
+        ObaRegion region = Application.get().getCurrentRegion();
+        return !(region == null || region.getContactEmail() == null);
+    }
+
+    @Override
+    public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
+        ReportTypeItem rti = (ReportTypeItem) getListView().getItemAtPosition(i);
+        if (getString(R.string.rt_infrastructure_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity();
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_stop_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_stop));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_arrival_problem).equals(rti.getTitle())) {
+            // Report bus stop issue
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_trip));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
```

Comment:
Can you please double-check that these analytics events fire when they are called after opening the new activity, or firing the intent for the IdeaScale website?  They should, but I would just like to confirm.